typescript/sdk/src/hook/CosmosNativeHookModule.ts

```diff
@@ -184,10 +184,11 @@ export class CosmosNativeHookModule extends HyperlaneModule<
       });
     }
 
-    if (config.owner && this.signer.account.address !== config.owner) {
+    if (this.signer.account.address !== config.owner) {
```

Comment:
should this be !eqAddress instead of just a !== comparison? ditto generally when we're comparing addresses